README.md

```diff
@@ -423,6 +423,24 @@ end
 <%= component "button", label: "Sign in", url: sign_in_path %>
 ```
 
+### Validation
```

Comment:
Perhaps name "Attribute validations" and move below the "Attribute overrides" section?

Reply:
Sure - will do.